test/expectations.jl

```diff
@@ -0,0 +1,88 @@
+@testset "expected_loglikelihood" begin
```

Comment:
```suggestion
@testset "expectations" begin
```
match source file name